src/OrchardCore.Modules/OrchardCore.Setup/Controllers/SetupController.cs

```diff
@@ -163,6 +163,13 @@ public async Task<ActionResult> IndexPOST(SetupViewModel model)
             else
             {
                 setupContext.DatabaseProvider = model.DatabaseProvider;
+
+                if (model.DatabaseProvider == "Sqlite")
+                {
+                    const string defaultDatabaseName = "OrchardCore.db";
+                    model.ConnectionString = $"Data Source={defaultDatabaseName};Cache=Shared";
```

Comment:
We should be able to set a relative path like `./orchardcore.db`

Reply:
I'm not sure if the relative path accepted in SQLite connectionString

Reply:
I assume there is still no way to change the value created by the setup. How would someone do that?
And if we use the current text box like for the other databases, there is a risk to access folders outside of the app_data. If it was done in the appsettings directly.

I think using a relative path is still dangerous. We should ensure that only a filename can be provided, and force the location.

Reply:
I think no need to provide a filename either, it could be done automatically, like what we have today, but for backward compatibility the user may change the filename from the `appsettings.json` which located into a particular tenant